lib/abr/simple_abr_manager.js

```diff
@@ -345,8 +348,19 @@ shaka.abr.SimpleAbrManager = class {
    */
   getBandwidthEstimate() {
     const defaultBandwidthEstimate = this.getDefaultBandwidth_();
```

Comment:
You should remove the `getDefaultBandwidth_` function and just move the code from there into this function.

That way you wouldn't need to do that unreliable `this.config_.defaultBandwidthEstimate !== defaultBandwidthEstimate` check below, and could just add a boolean to track whether `navigator.connection.downlink` has been used or not.